test/testsuite.go

```diff
@@ -0,0 +1,34 @@
+package testutils
+
+ import (
+	"log"
+	"os"
+
+ 	"github.com/codeready-toolchain/registration-service/pkg/configuration"
+	"github.com/stretchr/testify/suite"
+)
+
+ // UnitTestSuite is the base test suite for unit tests.
+type UnitTestSuite struct {
+	suite.Suite
+	ConfigRegistry *configuration.Registry
```

Comment:
Can we call it just `Config`?
```suggestion
	Config *configuration.Registry
```

Reply:
yup! fixed here: 1b2cb31ed23212e813dc3dee0fa909e4a0d63551